ArduPlane/Log.cpp

```diff
@@ -102,13 +102,19 @@ void Plane::Log_Write_Control_Tuning()
         synthetic_airspeed = logger.quiet_nan();
     }
 
+    int16_t pitch = ahrs.pitch_sensor - g.pitch_trim * 100;
+#if HAL_QUADPLANE_ENABLED
+    if (quadplane.in_vtol_mode()) {
+    pitch = ahrs.pitch_sensor - quadplane.ahrs_trim_pitch * 100;
+    }
```

Comment:
```suggestion
    if (quadplane.show_vtol_view()) {
        pitch = quadplane.ahrs_view->pitch_sensor;
    }
```

Reply:
For Quadplane we use the ahrs view which includes pitch trim already.

Reply:
changed....graphs in flight are the same, so I am happy